### PR TITLE
[CI] fix Substrate rustdoc link checker

### PR DESCRIPTION
**Pull request by jakehemmerle**
Don't fail building if the latest Substrate version doesn't have rustdoc links for it. Could be more concise (code was pretty wet to start) but master should only fail now when there are real issues.

### CODE REVIEW

**Comment by jakehemmerle** [comment]
should fix substrate rustdoc link updater in #2294

File: scripts/update-substrate-links.mjs [hunk 1]
```diff
@@ -10,30 +15,28 @@ import fs from "fs";
...
+    
+    while (!pass) {
+      let testUrl = `http://substrate.dev/rustdocs/${tagTemp}/sc_service/index.html`;
+      try {
+        const res = await axios.get(testUrl);
+        if (res.status === 200) pass = true;
+      } catch (err) {
+        tagTemp = tagTemp.slice(0, -1);
+        if (tagTemp.length === 0) {
+          console.log(`NOTE: '${tag}' and variants of its prefixes are not valid Substrate rustdoc version numbers. No links to update!`);
```
Comment (emresurmeli) [inline scripts/update-substrate-links.mjs:31]: We can log the err as well

**Review by salmad3 — COMMENTED** [review]
tested and LGTM

File: scripts/update-substrate-links.mjs (hunk 1 above)
Comment (jakehemmerle) [reply scripts/update-substrate-links.mjs:31]: I don't think it's noteworthy, every error is always a 404 and will pollute the logs with every prefix try. If we hit that console.log, it means we've tried every prefix

**Pull request merged**